x-pack/plugins/cases/public/components/case_view/components/case_view_activity.test.tsx

```diff
@@ -222,117 +230,132 @@ describe('Case View Page activity tab', () => {
+    expect(screen.queryByTestId('user-actions-list')).not.toBeInTheDocument();
   });
 
   it('should show a loading when updating severity ', async () => {
     useOnUpdateFieldMock.mockReturnValue({ isLoading: true, loadingKey: 'severity' });
 
-    const result = appMockRender.render(<CaseViewActivity {...caseProps} />);
+    appMockRender.render(<CaseViewActivity {...caseProps} />);
 
     expect(
-      result
-        .getByTestId('case-severity-selection')
-        .classList.contains('euiSuperSelectControl-isLoading')
+      (await screen.findByTestId('case-severity-selection')).classList.contains(
+        'euiSuperSelectControl-isLoading'
+      )
     ).toBeTruthy();
   });
 
   it('should not show a loading for severity when updating tags', async () => {
     useOnUpdateFieldMock.mockReturnValue({ isLoading: true, loadingKey: 'tags' });
 
-    const result = appMockRender.render(<CaseViewActivity {...caseProps} />);
+    appMockRender.render(<CaseViewActivity {...caseProps} />);
 
     expect(
-      result
-        .getByTestId('case-severity-selection')
-        .classList.contains('euiSuperSelectControl-isLoading')
+      (await screen.findByTestId('case-severity-selection')).classList.contains(
+        'euiSuperSelectControl-isLoading'
+      )
     ).not.toBeTruthy();
   });
 
   it('should not render the assignees on basic license', () => {
+    useCasesFeaturesMock.mockReturnValue({
+      ...useGetCasesFeaturesRes,
+      caseAssignmentAuthorized: false,
+    });
+
     appMockRender = createAppMockRenderer({ license: basicLicense });
 
-    const result = appMockRender.render(<CaseViewActivity {...caseProps} />);
-    expect(result.queryByTestId('case-view-assignees')).toBeNull();
+    appMockRender.render(<CaseViewActivity {...caseProps} />);
+    expect(screen.queryByTestId('case-view-assignees')).not.toBeInTheDocument();
   });
 
   it('should render the assignees on platinum license', async () => {
     appMockRender = createAppMockRenderer({ license: platinumLicense });
 
-    const result = appMockRender.render(<CaseViewActivity {...caseProps} />);
-    expect(result.getByTestId('case-view-assignees')).toBeInTheDocument();
+    appMockRender.render(<CaseViewActivity {...caseProps} />);
+    expect(await screen.findByTestId('case-view-assignees')).toBeInTheDocument();
 
     await waitForComponentToUpdate();
   });
 
   it('should not render the connector on basic license', () => {
+    useCasesFeaturesMock.mockReturnValue({
+      ...useGetCasesFeaturesRes,
+      pushToServiceAuthorized: false,
+    });
+
     appMockRender = createAppMockRenderer({ license: basicLicense });
 
-    const result = appMockRender.render(<CaseViewActivity {...caseProps} />);
-    expect(result.queryByTestId('case-view-edit-connector')).toBeNull();
+    appMockRender.render(<CaseViewActivity {...caseProps} />);
+    expect(screen.queryByTestId('case-view-edit-connector')).toBeNull();
```

Comment:
nit: to be consistent, maybe
```suggestion
    expect(screen.queryByTestId('case-view-edit-connector')).not.toBeInTheDocument();
```
?